<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Plot3DSurface</class>
 <widget class="QWidget" name="Plot3DSurface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>974</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="title">
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="plot_widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Theme</string>
       </property>
       <property name="buddy">
        <cstring>theme</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="theme"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Gradient</string>
       </property>
       <property name="buddy">
        <cstring>gradient</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="gradient"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Shadow</string>
       </property>
       <property name="buddy">
        <cstring>shadow</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="shadow"/>
     </item>
     <item>
      <widget class="QCheckBox" name="wireframe">
       <property name="text">
        <string>Show wireframe</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save_image">
       <property name="text">
        <string>Save image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save_text">
       <property name="text">
        <string>Save text</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="clear">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
